bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/DataClientWrapper.java

```diff
@@ -56,6 +59,13 @@
    */
   ApiFuture<List<KeyOffset>> sampleRowKeysAsync(String tableId);
 
+  /** Reads a single row, if the row not found then returns an empty {@link Result}. */
+  ApiFuture<Result> readRowAsync(String tableId, ByteString rowKey);
```

Comment:
I dont think we need this. Lets just stick with the nullable filter below. I'd like to keep this abstraction layer narrow